### PR TITLE
Add CORS support to blackd

### PR DESCRIPTION
**Pull request by tuchandra**
See #622. This is my first time contributing to a project that uses pipenv, so please let me know if there's something I did wrong.

Additionally, I'm not sure how to add a test for this. I'd expect it to test that a request sent from a client on a different host works correctly, but I read through the docs for the aiohttp [TestClient](https://docs.aiohttp.org/en/stable/testing.html?highlight=testing#test-client) and couldn't come up with anything. Setting the host param in the constructor for TestClient `async with TestClient(TestServer(app), host=some_string)` gives a TypeError about the host parameter being invalid.

### CODE REVIEW

**Comment by tuchandra** [comment]
@zsol, when you get a chance could you give this a look?

File: blackd.py [hunk 1]
```diff
@@ -35,7 +45,18 @@ def main(bind_host: str, bind_port: int) -> None:
...
     executor = ProcessPoolExecutor()
-    app.add_routes([web.post("/", partial(handle, executor=executor))])
+
+    cors = aiohttp_cors.setup(app)
+    resource = cors.add(app.router.add_resource("/"))
+    cors.add(
+        resource.add_route("POST", partial(handle, executor=executor)),
+        {
+            "*": aiohttp_cors.ResourceOptions(
+                allow_headers=(*BLACK_HEADERS, "Content-Type")
```
Comment (zsol) [inline blackd.py]: We'll probably also want to expose some headers. At least `Content-Type`, but maybe just `*`?

Comment (tuchandra) [reply blackd.py]: Fine with me; fixed in the most recent commit.

**Comment by zsol** [comment]
Thanks!

**Pull request merged**